doc/classes/Array.xml

```diff
@@ -197,6 +197,25 @@
 			</argument>
 			<description>
 				Finds the index of an existing value (or the insertion index that maintains sorting order, if the value is not yet present in the array) using binary search and a custom comparison method. Optionally, a [code]before[/code] specifier can be passed. If [code]false[/code], the returned index comes after all existing entries of the value in the array. The custom method receives two arguments (an element from the array and the value searched for) and must return [code]true[/code] if the first argument is less than the second, and return [code]false[/code] otherwise.
+				[codeblock]
+				func cardinal_to_algebraic(a):
+				    match a:
+				        "one":
+				            return 1
+				        "two":
+				            return 2
+				        "three":
+				            return 3
+				        "four":
+				            return 4
+				
+				func compare(a,b):
```

Comment:
I tried to get you to change this with 3 suggestions so far... so let's try a more detailed approach.

The Godot class reference follows the GDScript style guide, and this style guide requires a space after a comma: https://docs.godotengine.org/en/stable/getting_started/scripting/gdscript/gdscript_styleguide.html#whitespace

Reply:
I added your suggestions. Somehow my git repo is messed up. Maybe I am tired.